examples/metadata.rs

```diff
@@ -86,5 +88,8 @@ fn main() {
     let topic = matches.value_of("topic");
     let fetch_offsets = matches.is_present("offsets");
 
-    print_metadata(brokers, topic, (timeout * 1000f32) as i32, fetch_offsets);
+    print_metadata(brokers,
+                   topic,
+                   Some(Duration::from_millis((timeout * 1000f32) as u64)),
```

Comment:
I think we could just change the command line the parameter to milliseconds (directly in `u64`) and remove the conversion here.